common/src/main/java/org/openmrs/analytics/JdbcConnectionUtil.java

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.openmrs.analytics;
+
+import java.beans.PropertyVetoException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import com.mchange.v2.c3p0.ComboPooledDataSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JdbcConnectionUtil {
+	
+	private static final Logger log = LoggerFactory.getLogger(JdbcConnectionUtil.class);
+	
+	private static ComboPooledDataSource comboPooledDataSource;
+	
+	private String jdbcDriverClass;
+	
+	private String jdbcUrl;
+	
+	private String dbUser;
+	
+	private String dbPassword;
+	
+	private Integer initialPoolSize;
+	
+	private Integer dbcMaxPoolSize;
+	
+	JdbcConnectionUtil(String jdbcDriverClass, String jdbcUrl, String dbUser, String dbPassword)
+	        throws PropertyVetoException {
+		
+		this.jdbcDriverClass = jdbcDriverClass;
+		this.jdbcUrl = jdbcUrl;
+		this.dbUser = dbUser;
+		this.dbPassword = dbPassword;
+	}
+	
+	JdbcConnectionUtil(String jdbcDriverClass, String jdbcUrl, String dbUser, String dbPassword, int dbcMaxPoolSize)
+	        throws PropertyVetoException {
+		
+		this.jdbcDriverClass = jdbcDriverClass;
+		this.jdbcUrl = jdbcUrl;
+		this.dbUser = dbUser;
+		this.dbPassword = dbPassword;
+		this.dbcMaxPoolSize = dbcMaxPoolSize;
+		this.initialPoolSize = 10;
+	}
+	
+	public Statement createStatement() throws SQLException, PropertyVetoException {
+		Connection con = getConnectionObject().getConnection();
+		return con.createStatement();
+	}
+	
+	public ComboPooledDataSource getConnectionObject() throws PropertyVetoException {
+		comboPooledDataSource = new ComboPooledDataSource();
+		comboPooledDataSource.setDriverClass(this.jdbcDriverClass);
+		comboPooledDataSource.setJdbcUrl(this.jdbcUrl);
+		comboPooledDataSource.setUser(this.dbUser);
+		comboPooledDataSource.setPassword(this.dbPassword);
+		if (this.dbcMaxPoolSize != null || this.initialPoolSize != null) {
+			comboPooledDataSource.setMaxPoolSize(this.dbcMaxPoolSize);
+			comboPooledDataSource.setInitialPoolSize(10);
```

Comment:
i had set it like this because its the value that was set according to this pr https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/commit/25975894d9280054f9373b29a589fe8d72097f23

Reply:
It is fine to hard-code `10`, My question was that you have the `initialPoolSize` but you are not using it, why? Either use `10` directly and remove the `initialPoolSize` property OR avoid hard-coding `10` here and use `initialPoolSize` to `setInitialPoolSize()`.